primo/linking/intent_filters.pyx

```diff
@@ -170,6 +170,7 @@ cdef class IntentFilter(object):
     """
 
     return (self.actions and len(self.actions) >= 1
+            and hasattr(IntentFilter, '_ACTION_MAIN')
```

Comment:
Actually, the bug here is probably caused by the fact that _ACTION_MAIN is not currently an attribute of IntentFilter. Instead of IntentFilter._ACTION_MAIN, it should just be _ACTION_MAIN.